src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        class="card__image"
+        alt="APPLE A1419 iMac 27"
+      >
+      <div class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="stars">
+        <div class="stars__elem stars__elem--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="stars__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-word">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+      <div class="card__button-buy">
+        <a
+          href="https://mate.academy/learn?module=css-advanced&course=fullstack"
+          class="card__text-button-buy"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
it's a redundant `<div>` wrapper. Apply style for <a>
```suggestion
        <a
          href="https://mate.academy/learn?module=css-advanced&course=fullstack"
          class="card__text-button-buy"
          data-qa="hover"
        >
          BUY
        </a>
```

Reply:
done